src/dataDisplay/dataGrid/dataGrid.jsx

```diff
@@ -1,43 +1,90 @@
 import ClassNames from 'classnames';
 import PropTypes from 'prop-types';
-import React, { useEffect } from 'react';
+import React from 'react';
 import { BEM_DATA_GRID } from '../../global/constants';
 import { SORTABLE_PROP_TYPES } from './dataGridConstants';
 import DataGridTable from './dataGridTable';
 import makeStyles from '../../styles/makeStyles';
 
 const propTypes = {
+    /**
+     * If <code>true</code>, the component will bleed to the edge of the parent component.
```

Comment:
Please just backticks instead of `<code>`.

Reply:
done